ts/screens/wallet/FirstTransactionSummaryScreen.tsx

```diff
@@ -0,0 +1,261 @@
+import {
+  Body,
+  Button,
+  Container,
+  Content,
+  H1,
+  H3,
+  Icon,
+  Left,
+  Right,
+  Text,
+  View
+} from "native-base";
+import * as React from "react";
+import { Image, StyleSheet } from "react-native";
+import { Grid, Row } from "react-native-easy-grid";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import AppHeader from "../../components/ui/AppHeader";
+import Modal from "../../components/ui/Modal";
+import PaymentSummaryComponent from "../../components/wallet/paymentSummary/PaymentSummaryComponent";
+import UpdatedPaymentSummaryComponent from "../../components/wallet/paymentSummary/UpdatedPaymentSummaryComponent";
+import I18n from "../../i18n";
+import variables from "../../theme/variables";
+
+type Props = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+}>;
+
+type State = Readonly<{
+  isModalVisible: boolean;
+  isAmountUpdated: boolean; // its value will depend on the match withe the amount on the notice and the remote value
+  amount: string;
+  // The following data should be globally obtained as result of the transaction identification.
+  // Further discussions on thier availability is required
+  updatedAmount: string;
+  expireDate: string;
+  tranche: string;
+}>;
+
+const styles = StyleSheet.create({
+  padded: {
+    paddingRight: variables.contentPadding,
+    paddingLeft: variables.contentPadding
+  }
+});
+
+export class FirstTransactionSummaryScreen extends React.Component<
+  Props,
+  State
+> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      isModalVisible: false,
+      isAmountUpdated: true,
+      amount: "€ 199,00",
+      updatedAmount: "€ 215,00",
+      expireDate: "31/01/2018",
+      tranche: "unica"
+    };
+  }
+
+  private goBack() {
+    this.props.navigation.goBack();
+  }
+
+  /* depending on the comparison between the amount on the notice and the amount saved remotely by the lender
+   * it will be displayed a different component
+  */
+  private getSummary() {
+    if (this.state.isAmountUpdated === true) {
+      return (
+        // if the amount had been updated, it will be displayed and presented with a brief description
+        <UpdatedPaymentSummaryComponent
+          navigation={this.props.navigation}
+          amount={this.state.amount}
+          updatedAmount={this.state.updatedAmount}
+          expireDate={this.state.expireDate}
+          tranche={this.state.tranche}
+        />
+      );
+    } else {
+      return (
+        <PaymentSummaryComponent
+          navigation={this.props.navigation}
+          amount={this.state.amount}
+          expireDate={this.state.expireDate}
+          tranche={this.state.tranche}
+        />
+      );
+    }
+  }
+
+  public render(): React.ReactNode {
+    // The following data should be globally obtained as result of the transaction identification.
+    // Further discussions on thier availability is required
+    const NOMEAVVISO: string = "Tari 2018";
```

Comment:
you want to create a type for collecting all these attributes :)

Reply:
ideally, we should reuse the type generated from a definition provided by the backend APIs (and the pagopaproxy APIs) - cc @gunzip @fgiobergia @LsKoder